mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -41,6 +41,8 @@
     <!-- Browser menu button that opens the settings menu -->
     <string name="browser_menu_settings">Settings</string>
     <!-- Browser menu button that opens a user's library -->
+    <string name="browser_menu_your_library">Your Library</string>
+    <!-- Deprecated: Browser menu button that opens a user's library -->
     <string name="browser_menu_library">Library</string>
```

Comment:
This should be removed entirely from the file so that it doesn't appear in Pontoon